internal/shim-sgx/src/handler.rs

```diff
@@ -447,20 +450,23 @@ impl<'a> SyscallHandler for Handler<'a> {
         target_info.mrenclave.copy_from_slice(&ti[0..32]);
         target_info.attributes = att;
         let data = ReportData([0u8; 64]);
-        let _report: Report = unsafe { target_info.get_report(&data) };
+        let report: Report = unsafe { target_info.get_report(&data) };
 
         // Request Quote from host
-        // TODO: Send a real Report. See https://github.com/enarx/enarx-keepldr/issues/92.
-        let tmp_report = [0u8; 512];
+        let report_len = core::mem::size_of::<Report>();
+        let report_slice = &[report];
+        let (_, report_bytes, _) = unsafe { report_slice.align_to::<u8>() };
+        assert_eq!(report_bytes.len(), report_len);
```

Comment:
The `assert_eq` should probably be a `debug_assert_eq`, because we can assume the behavior doesn't change in   release version.

I give you, this is a lot less code to write and read, than:
```rust
        let report_bytes = unsafe {
            core::slice::from_raw_parts(
                report_slice.as_ptr() as *const _ as *const u8,
                core::mem::size_of::<Report>(),
            )
        };
```
although I first wondered why the align to `u8` ... but yeah, shorter to write :+1: